src/app/modules/shared/shared.module.ts

```diff
@@ -54,6 +56,9 @@ import { CreateAccountsFormComponent } from './components/create-accounts-form/c
     ImportAccountsFormComponent,
     CreateAccountsFormComponent,
   ],
+  providers: [
+    BreadcrumbService,
```

Comment:
Since this service is scoped only to the shared module, no need to import it in the top-level app.module.ts. Importing it here is all we need